src/components/Scorecards/ScorecardDetailsPage/ScorecardDetails.tsx

```diff
@@ -55,9 +60,43 @@ export const ScorecardDetails = ({
     return scores.filter(filter);
   }, [scores, filter]);
 
+  const copyCSV = useCallback(
```

Comment:
nit: we tend to use strict camel casing: https://github.com/cortexapps/brain-app/blob/master/apps/%40cortexapps/brain-app/BEST_PRACTICES.md#use-strict-camel-casing

```suggestion
  const copyCsv = useCallback(
```

Reply:
no longer existing